src/app/main/api-gest-console/modules/aig-standard/components/ipp-lot-type-dialog-page/ipp-lot-type-dialog-page.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="dialog-content-wrapper">
+    <mat-toolbar matDialogTitle class="mat-accent m-0">
+        <mat-toolbar-row fxLayout="row" fxLayoutAlign="space-between center">
+            <span class="title dialog-title">Add Ipp LotType</span>
+            <button mat-icon-button (click)="matDialogRef.close()" aria-label="Close dialog">
+                <mat-icon>close</mat-icon>
+            </button>
+        </mat-toolbar-row>
+    </mat-toolbar>
+
+    <div mat-dialog-content class="p-24 pb-0 m-0" fusePerfectScrollbar>
+        <aig-ipp-lot-type-new-update-form></aig-ipp-lot-type-new-update-form>
```

Comment:
you need pass the param for populate in case of update see [this example](https://github.com/EneaSys/aig_console/blob/develop/src/app/main/api-gest-console/modules/aig-generic/components/eopoo-type-new-update-modal/eopoo-type-new-update-modal.component.html)